shared/src/androidMain/kotlin/org/hyperskill/app/purchases/domain/AndroidPurchaseManager.kt

```diff
@@ -0,0 +1,110 @@
+package org.hyperskill.app.purchases.domain
+
+import android.app.Activity
+import android.app.Application
+import com.revenuecat.purchases.LogLevel
+import com.revenuecat.purchases.PurchaseParams
+import com.revenuecat.purchases.Purchases
+import com.revenuecat.purchases.PurchasesConfiguration
+import com.revenuecat.purchases.PurchasesErrorCode
+import com.revenuecat.purchases.PurchasesException
+import com.revenuecat.purchases.PurchasesTransactionException
+import com.revenuecat.purchases.awaitCustomerInfo
+import com.revenuecat.purchases.awaitGetProducts
+import com.revenuecat.purchases.awaitLogIn
+import com.revenuecat.purchases.awaitLogOut
+import com.revenuecat.purchases.awaitPurchase
+import com.revenuecat.purchases.models.StoreProduct
+import org.hyperskill.app.BuildConfig
+import org.hyperskill.app.purchases.domain.model.AndroidPurchaseParams
+import org.hyperskill.app.purchases.domain.model.PlatformPurchaseParams
+import org.hyperskill.app.purchases.domain.model.PurchaseManager
+import org.hyperskill.app.purchases.domain.model.PurchaseResult
+
+class AndroidPurchaseManager(
+    private val application: Application,
+    private val isDebugMode: Boolean
+) : PurchaseManager {
+    override fun setup() {
+        if (!Purchases.isConfigured) {
+            Purchases.logLevel = if (isDebugMode) LogLevel.DEBUG else LogLevel.INFO
```

Comment:
I agree that for debugging issues LogLevel.DEBUG in debug builds will be helpful, but I don't understand how we can use logs in release builds, where logs will be logged?

1. https://sdk.revenuecat.com/android/7.4.0/purchases/com.revenuecat.purchases/-purchases/-companion/log-level.html
2. https://sdk.revenuecat.com/android/7.4.0/purchases/com.revenuecat.purchases/-purchases/-companion/log-handler.html

_Debug logs will provide detailed log output in Xcode or LogCat for what is going on behind the scenes and should be the first thing you check if your app is behaving unexpectedly, and also to confirm there aren't any unhandled warnings or errors._

Reply:
@ivan-magda 
> how we can use logs in release builds, where logs will be logged?

You are right, we can't collect logs in release mode.

However, LogLevel  in RevenueCat can not be completely disabled. This field is not nullable.